pkg/daemon/daemon.go

```diff
@@ -2812,73 +2813,81 @@ func cipherOrder() []uint16 {
 	return append(first, second...)
 }
 
-type rpmOstreeStatus struct {
-	Deployments []struct {
-		RequestedPackages []string `json:"requested-local-packages"`
-	} `json:"deployments"`
+type Deployment struct {
+	Booted                             bool     `json:"booted"`
+	RequestedPackages                  []string `json:"requested-packages"`
+	RequestedLocalPackages             []string `json:"requested-local-packages"`
+	RequestedLocalFileoverridePackages []string `json:"requested-local-fileoverride-packages"`
+	RequestedBaseRemovals              []string `json:"requested-base-removals"`
+	RequestedBaseLocalReplacements     []string `json:"requested-base-local-replacements"`
+	RequestedModules                   []string `json:"requested-modules"`
```

Comment:
After some slack discussion I think we don't need to check for Modules. I think it's fine to remove this and track everything else.